### PR TITLE
tiva/cc13x2_cc26x2: Fix nxstyle errors

### PR DESCRIPTION
**Pull request by hartmannathan**
## Summary

arch/arm/src/tiva/hardware/cc13x2_cc26x2/cc13x2_cc26x2_flash.h:

- Fix nxstyle errors.

## Impact

Removes nxstyle errors.

## Testing

nxstyle

### CODE REVIEW

**Review by Ouss4 — APPROVED** [review]

**Review by btashton — COMMENTED** [review]
Ugh I had made these comments but they apparently we're stuck pending. Too late now

File: arch/arm/src/tiva/hardware/cc13x2_cc26x2/cc13x2_cc26x2_flash.h [hunk 1]
```diff
@@ -993,10 +1002,15 @@
 #define FLASH_FSM_EXECUTE_SUSPEND_NOW_MASK   (15 << FLASH_FSM_EXECUTE_SUSPEND_NOW_SHIFT)
 
 /* TIVA_FLASH_FSM_SECTOR1 (32-bit value) */
+
```
[the remark this answers is not in the document]
Comment (hartmannathan) [reply arch/arm/src/tiva/hardware/cc13x2_cc26x2/cc13x2_cc26x2_flash.h:1005]: Oops it looks like that was intended to be a single section separator for several similar registers. Sorry. It's been merged already. I can make another PR to make them a single comment.

[the remark this answers is not in the document]
Comment (hartmannathan) [reply arch/arm/src/tiva/hardware/cc13x2_cc26x2/cc13x2_cc26x2_flash.h:1005]: Done. See [PR-2250](https://github.com/apache/incubator-nuttx/pull/2250)

[the remark this answers is not in the document]
Comment (btashton) [reply arch/arm/src/tiva/hardware/cc13x2_cc26x2/cc13x2_cc26x2_flash.h:1005]: No worries. It was my fault, the GitHub mobile app changed the default to make it a pending review and I forgot to submit it at the end.

Thanks for doing all of these.

**Pull request merged**